parlai/crowdsourcing/tasks/model_chat/README.md

```diff
@@ -0,0 +1,69 @@
+# Model chat crowdsourcing task
+
+This task will collect conversations between a human and a model. After each response by the model, the human will optionally be prompted to annotate the model's response by selecting checkboxes that represent customizable attributes of that response.
+
+**NOTE**: See [parlai/crowdsourcing/README.md](https://github.com/facebookresearch/ParlAI/blob/master/parlai/crowdsourcing/README.md) for general tips on running `parlai.crowdsourcing` tasks, such as how to specify your own YAML file of configuration settings, how to run tasks live, how to set parameters on the command line, etc.
+
+## Launching
+
+Call `run.py` to run this task with the default parameters, as set by `conf/example.yaml`. Some parameters that you can adjust include where to save data, lists of workers to soft-block, the maximum response time, etc.
+
+The models used for chat should be placed in a single base folder, specified by the `mephisto.blueprint.base_model_folder` flag. Each model file should be named `model` and should be located in its own subfolder, and all options used when running the model should be listed in a JSON file named `model.opt` in that subfolder. For example, suppose your base model folder is `~/ParlAI/data/models/`, and you wish to run 10 conversations of the `blender_90M` model. Suppose that the contents of `~/ParlAI/data/models/` looks like this:
+```
+blender/blender_3B:
+model
+model.dict
+model.dict.opt
+model.opt
+
+blender/blender_90M:
+model
+model.dict
+model.dict.opt
+model.opt
+```
+You might then run the following command:
+```
+python parlai/crowdsourcing/tasks/model_chat/run.py \
+mephisto.blueprint.base_model_folder=~/ParlAI/data/models \
+mephisto.blueprint.conversations_needed_string=\"blender/blender_90M:10\"
+```
+
+## Passing in task config files
+
+The following flags can be passed in to specify filepaths for overriding the text shown to the workers and the settings of the annotation categories. If they are not specified, the defaults in the `task_config/` folder will be used.
+- `mephisto.blueprint.annotations_config_path`: JSON file configuring annotation categories. Set this flag to `""` to disable annotation of model responses.
```

Comment:
Yes, that's fair - I've just changed the default val of this param to `""`. However, I've set this param to `${task_dir}/task_config/annotations_config.json` in `example.yaml`, because I think it'd be useful for users to see how annotation works